src/main/main/java/Duke.java

```diff
@@ -46,16 +46,38 @@ public static void main(String[] args) {
                 list[taskNum-1].markAsDone();
                 System.out.println("____________________________________________________________\n"
                         + "Nice! I've marked this task as done:");
-                System.out.println(" [" + list[taskNum-1].getStatusIcon() + "] " + list[taskNum-1].description);
+                System.out.println(" " + list[taskNum-1].toString());
                 System.out.println("____________________________________________________________\n");
                 continue;
```

Comment:
I am not sure that what do you mean skip the for loop for using the "continue" here.